### PR TITLE
TestReport

### PR DESCRIPTION
**Pull request by MariaJose319**
image

### CODE REVIEW

**Review by manuelq12 — DISMISSED** [review]
🏅

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "mocha": "^10.0.0"
   },
   "dependencies": {
-    "axios": "^0.27.2"
+    "axios": "^0.27.2",
+    "mochawesome": "^7.1.3"
```
Comment (manuelq12) [inline package.json]: As it is a Dev Dependency, it shouldn't be here
Mochawesome must be in the devDependencies

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (manuelq12) [reply package.json]: `npm uninstall mochawesome`
    `npm install mochawesome -D`

Comment (MariaJose319) [reply package.json]: image
I've already changed it. Thanks.

**Review by manuelq12 — APPROVED** [review]
LGTM 👍

**Pull request merged**